source/features/linkify-branch-refs.tsx

```diff
@@ -17,7 +17,7 @@ function inPR() {
 
 	// Find the URLs first, some elements don't have titles
 	const urls = new Map<string, string>();
-	for (const el of select.all('.commit-ref[title], .base-ref[title], .head-ref[title]')) {
+	for (const el of select.all<HTMLElement>('.commit-ref[title], .base-ref[title], .head-ref[title]')) {
```

Comment:
Used the generic `HTMLElement` as I have no idea what kind of element this is supposed to be. I would guess a `<div />`, but not certain.

Reply:
I'd stick to `HTMLElement`